Implement docs feedback from heuristic walkthrough (#223)
AlexsLemonade/scpca-downstream-analyses: README.md

To run the core downstream analyses workflow on your own sample data, you will need the following:

1. A local installation of Snakemake and either R or conda (see more on this in the ["how to install the core downstream analyses workflow" section](#how-to-install-the-core-downstream-analyses-workflow))
2. Single-cell gene expression data stored as `SingleCellExperiment` objects stored as RDS files (see more on this in the ["Input data format" section](#input-data-format))

> 3. A project metadata tab-separated value (TSV) file containing relevant information about your data necessary for processing (see more on this in the ["Metadata file format" section](#metadata-file-format))

Reviewer: Maybe we can link the example metadata file here as well. Thinking about users who are coming back to the documentation after while. It would be nice for them to find what they might want without too much bouncing from section to section.

Could we also mention need to a config file here and link to the example file?

4. A mitochondrial gene list that is compatible with your data (see more on this in the ["Running the workflow" section](#running-the-workflow))

<!-- cmd -->

> There are two expected output files thay will be associated with each provided `SingleCellExperiment` object and `library_id`: a processed `SingleCellExperiment` object containing normalized data and clustering results, and a summary html report detailing the filtering of low quality cells, dimensionality reduction, and clustering that was performed within the workflow. See the [expected output section](#expected-output) for more information on these output files.

Reviewer: ```suggestion
**Output Files**
There are two expected output files thay will be associated with each provided `SingleCellExperiment` object and `library_id`: 
- A processed `SingleCellExperiment` object containing normalized data and clustering results
- A summary HTML report detailing:
    - Filtering of low quality cells
    - Dimensionality reduction
    - Clustering that was performed within the workflow. 

See the [expected output section](#expected-output) for more information on these output files.
```

Breaking it up so it is easier to scan. Please correct phrasing to fit this format, if needed.

You can install Snakemake by following the [instructions provided in Snakemake's docs](https://snakemake.readthedocs.io/en/v7.3.8/getting_started/installation.html#installation-via-conda-mamba).

> As described in the Snakemake instructions, the recommended way to install snakemake is using the conda package manager. 

Reviewer: Can we make these two line more concise? Perhaps something like this:

```suggestion
Snakemake recommends installing it is using the conda package manager. 
```

> Therefore, you will want to first install [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html) (we recommend the Miniconda installation).

Reviewer: ```suggestion
Here are the instructions to install [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html) 
We recommend the Miniconda installation.
```

You can install Snakemake by following the [instructions provided in Snakemake's docs](https://snakemake.readthedocs.io/en/v7.3.8/getting_started/installation.html#installation-via-conda-mamba).

Snakemake recommends installing it using the conda package manager. 
Here are the instructions to [install conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html) 
We recommend the Miniconda installation.

> After installing conda has been installed, you can follow the steps below to set up the bioconda and conda-forge channels and install Snakemake in an isolated environment:

Reviewer: Whoop rephrasing to smooth it out. 

```suggestion
After installing conda, you can follow the steps below to set up the bioconda and conda-forge channels and install Snakemake in an isolated environment:
```

- `filepath`, the full path to the RDS file containing the pre-processed `SingleCellExperiment` object.
Each library ID should have a unique `filepath`.

|[View Example Metadata File](https://github.com/AlexsLemonade/scpca-downstream-analyses/blob/main/project-metadata/example-library-metadata.tsv)|
|---|

## Running the workflow

> We have provided a snakemake [configuration file](https://snakemake.readthedocs.io/en/stable/snakefiles/configuration.html), [`config.yaml`](https://github.com/AlexsLemonade/scpca-downstream-analyses/blob/main/config.yaml) which sets the defaults for all parameters needed to run the workflow.

Reviewer: ```suggestion
We have provided an example [snakemake configuration file](https://snakemake.readthedocs.io/en/stable/snakefiles/configuration.html), [`config.yaml`](https://github.com/AlexsLemonade/scpca-downstream-analyses/blob/main/config.yaml) which sets the defaults for all parameters needed to run the workflow.
```
